tests/tests.py

```diff
@@ -454,13 +454,61 @@ def test_res_comp_with_no_podpolicies(self):
                 removed = True
             else:
                 time.sleep(1)
+    
+    def test_appstatus_plugin(self):
 
+        def cleanup():
+            cmd = "cd ./hello-world"
```

Comment:
Rather than including ./hello-world directory in the tests folder, let's follow the pattern that we have used in test_application_update test, where we are referring to the files from the examples folder.

You can pretty much copy lines 90 till 117
https://github.com/cloud-ark/kubeplus/blob/b8ce8c1640c3373f6bbe9cc1ca2d8d78436f3626/tests/tests.py#L90

https://github.com/cloud-ark/kubeplus/blob/b8ce8c1640c3373f6bbe9cc1ca2d8d78436f3626/tests/tests.py#L117